contents/blog/sunsetting-helm-support-posthog.md

```diff
@@ -0,0 +1,62 @@
+---
+date: 2023-02-08
+title: "Sunsetting Kubernetes support for PostHog"
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: ["tim-glaser"]
+featuredImage: ../images/blog/k8s-sunset/posthog-bye-kubernetes.png
+featuredImageType: full
+category: PostHog news
+tags:
+  - Product updates
+---
+
+We're sunsetting support for our Kubernetes deployment for PostHog. Because we're an open source company, I want to be transparent about what this change means, and why we're making it. 
+
+In this blog post I'll explain:
+
+1. Why we're making this important change
+2. How this impacts existing self-hosted customers
+3. The steps we're taking to support impacted customers
+
+> **Important:** Right now, _about 3.5% of our users use Kubernetes_, so this change only affects a small subsection of our users. This decision **doesn't impact open source users** on Docker Compose deployments (also known as the "hobby" deployment). We remain totally committed to being open source and are actively exploring ways to make that experience even better.
+
+## Why are we doing this?
+
+PostHog has grown incredibly fast over the last three years. Last year, [we grew revenue 6x](https://posthog.com/blog/2022-review), we're default alive and will likely be profitable in 2023.
```

Comment:
I'd like to add some stats that aren't just revenue. @jamesefhawkins any you're happy talking about